Teensy/src/sdCard/sdCard.cpp

```diff
@@ -5,11 +5,135 @@
  * Detailed description, etc.
  */
 
+
 #include <Arduino.h>
+#include <SD.h>
+#include <SPI.h>
+#include <TimeLib.h>
+#include <DS1307RTC.h>
+
 #include "sdCard.h"
 
+File dataFile;
+String timeStamp;
+char * fileName;
+const int chipSelect = BUILTIN_SDCARD;
+
+
 extern bool sendDataToSdCard(sensorData_s *pSensorData)
 {
-    // Send data to SD card
+
+fileName = (char*) timeStamp.c_str();
+dataFile = SD.open(fileName, FILE_WRITE);
+
+  //if there was no problem opening the file => write to it
+  if (dataFile) {
+    Serial.print("writing to");
+    Serial.println(fileName);
```

Comment:
I had a go at this but I wasn't sure if it would work with ought converting both to (String) dtype so I just left it as is for now. Something like: 

Serial.println((String)  "Writing to " + gFilename);

I think could work.

Reply:
Did you test the line I suggested? If it didn't work then I'm happy to leave it as two separate lines as before, instead of having to worry about string casting